src/main/room/main/G1HRPS/Room.java

```diff
@@ -43,21 +43,21 @@ public Room(RoomType room_type, float price, String room_number, BedSize bedSize
 	/**
 	 * Prints out general information of the room
 	 */
-	public void viewRoomDetails(){
-		System.out.println("room number: "+room_num_);
-		System.out.println("room type "+room_type_);
-		System.out.println("room price: "+room_price_);
-		System.out.println("bedsize: "+bedsize_);
-		System.out.println("wifi enabled: "+wifi_enabled_);
-		System.out.println("with view: "+w_view_);
-		System.out.println("with smoking: "+w_smoking_);
-		System.out.println("room status: "+status_);
-		System.out.println("guest staying: "+guest_list_);
+	public String toString(){
+		String output = "room number: "+room_num_ + 
+						"\nroom price: "+room_price_+
+						"\nbedsize: "+bedsize_+
+						"\nwifi enabled: "+wifi_enabled_+
+						"\nwith view: "+w_view_+
+						"\nwith smoking: "+w_smoking_+
+						"\nroom status: "+status_+
+						"\nguest staying: "+guest_list_
+		return output;					
 	}
 	/**
 	 * Prints out whether a room is available or not
 	 */
-	public void checkRoomAvailability() {
+	public void CheckRoomAvailability() {
```

Comment:
this function is not required as manager should be the one determining this. and there's no need for printing output.